packages/augur-core/source/contracts/trading/ZeroXTrade.sol

```diff
@@ -20,6 +21,8 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
 
     bool transferFromAllowed = false;
 
+    uint256 constant public TRADE_INTERVAL_VALUE = 10 ** 19; // Trade value of 10 DAI
```

Comment:
Something to keep in mind, if Maker does a global shutdown DAI will become pegged to ETH, not USD and "10 DAI" has potential to be worth a lot or a little.